cibyl/cli/query.py

```diff
@@ -35,6 +35,61 @@ class QueryType(IntEnum):
     """Retrieve data concerning builds and above."""
 
 
+class QuerySelector:
+    """Deduce the type of query performed from the cli argument considering
+    both core argument and plugin provided ones."""
+    query_selector_functions = []
```

Comment:
If this attribute is going to be accessed statically, you can make the whole class be so. There is no point in having multiple instances if they are all going to share the same thing.

Reply:
You're right, there is no need for an instance of the class, I've changed it